pkg/api/v1alpha2/types_config.go

```diff
@@ -26,26 +26,26 @@ type ImageSetConfigurationSpec struct {
 
 // Mirror defines the configuration for content types within the imageset.
 type Mirror struct {
-	// OCP defines the configuration for OCP and OKD content types.
-	OCP OCP `json:"ocp,omitempty"`
+	// Platform defines the configuration for OpenShift and OKD platform types.
+	Platform `json:"platform,omitempty"`
 	// Operators defines the configuration for Operator content types.
 	Operators []Operator `json:"operators,omitempty"`
 	// AdditionalImages defines the configuration for a list
 	// of individual image content types.
-	AdditionalImages []AdditionalImages `json:"additionalImages,omitempty"`
+	AdditionalImages []Image `json:"additionalImages,omitempty"`
 	// Helm define the configuration for Helm content types.
-	Helm Helm `json:"helm,omitempty"`
+	Helm `json:"helm,omitempty"`
```

Comment:
Any particular reason for this change?

Reply:
@soltysh Copy. I originally removed it because it was redundant, but I see that this diverged from the way the rest of the fields are specified. I put it back in place to keep the way the fields are specified through the API spec consistent.